argonOne/run.sh

```diff
@@ -185,13 +185,12 @@ until false; do
         fanPercent=100;
         ;;
     esac
-    fanPercentHex=$(printf '%x' ${fanPercent})
-    action "${curPosition}" "${fanPercent}" "${curPositionName}" "${fanPercentHex}" "${cpuTemp}" "${CorF}"
+    action "${fanPercent}" "${curPosition}" "${curPositionName}" "${cpuTemp}" "${CorF}"
```

Comment:
Here you had a bug. You passed a `curPosition` into `percent` and vice versa. This fixes the issue https://github.com/adamoutler/HassOSArgonOneAddon/issues/19